Extension/package.json

```diff
@@ -495,6 +495,16 @@
           "default": true,
           "description": "If true, snippets are provided by the language server.",
           "scope": "resource"
+        },
+        "C_Cpp.enhancedColorization": {
+          "type": "string",
+          "enum": [
+            "Enabled",
+            "Disabled"
+          ],
+          "default": "Enabled",
+          "description": "If enabled, code is colorized based on syntactic and semantic analysis.  This setting has no effect if IntelliSense is disabled.",
```

Comment:
I think "syntactic" should be "lexical" or "syntactic and semantic analysis" should be something like "IntelliSense results".